x/dex/keeper/msgserver/msg_server_place_orders.go

```diff
@@ -39,6 +37,10 @@ func (k msgServer) transferFunds(goCtx context.Context, msg *types.MsgPlaceOrder
 			Amount:  sdk.NewDec(fund.Amount.Int64()),
 		})
 	}
+	escrow := k.AccountKeeper.GetModuleAccount(ctx, types.ModuleName).GetAddress()
+	if err := k.BankKeeper.SendCoins(ctx, sender, escrow, msg.Funds); err != nil {
```

Comment:
@BrandonWeng @codchen should this be a deferred send instead of a normal send? otherwise wouldn't this message have a bottleneck on module account? when is the deferred send processed relative to when the funds are needed for endblock processing?